communities/edinburgh-tech-learners/community-assessment.md

```diff
@@ -0,0 +1,120 @@
+## Assessing Your Community's Needs
+
+<!--
+Fully understanding the needs of your community is a complex concept that will likely take lots of time, conversation, and observation.  Depending on your situation, you may already have a great idea for what your community needs. You may not, and that's OK. We will go over these questions and topics broadly during week 1, and discuss responses in more detail in week 2. Here are some things to keep in mind when assessing your community and hoping to grow it.
+
+Please write down your
+- Answers to these questions for current community
+- Answers to these questions for **an ideal version** of your current community
+- Ideas on how to improve in each category
+
+After you've completed these questions, define your community in several short paragraphs for discussion in week 2.
+-->
+
+### Definition
+> - How would you define your community?
+
+New programmers in Edinburgh who want to develop their skills past the "intro to _X_" stage.
+
+### Size
+> - How many people are currently in your community?
+> - How many would you _like_ to be in your community?
+> - Historically, has the group always been about the same size? If not, what brought on that change?
+
+The number of people my definition applies to is quite large — but there's no real community among them at the moment. I won't have too much input on that number, but what I _do_ want to have an impact on is how many people are coming together to improve their skills. (And maybe by doing that, I will be able to encourage some more people to learn and participate and grow the community slightly.)
```

Comment:
Solid. With this new ground you're treading, it's totally okay to not have an idea of how big that could/should be, and I like how you've reframed this question